DataFormats/ParticleFlowCandidate/interface/PFCandidatePhotonExtra.h

```diff
@@ -61,7 +61,7 @@ namespace reco {
     void addLCorrClusEnergy(float LCorrE);
     const std::vector<float>& LCorrClusEnergy() const {return LocalCorr_;}
 
-    void SetPFPhotonRes(float Res){Res=MVAResolution_;}
+    void SetPFPhotonRes(float Res){MVAResolution_ = Res;}
```

Comment:
Yes that is what i thought and thats why i didnt remove it completely. Is
the current fix alright?

On Mon, Apr 25, 2016 at 4:49 PM, Carl Vuosalo <notifications@github.com>
wrote:

> In DataFormats/ParticleFlowCandidate/interface/PFCandidatePhotonExtra.h
> <https://github.com/cms-sw/cmssw/pull/14220#discussion_r60995735>:
>
> > @@ -61,7 +61,7 @@ namespace reco {
> >      void addLCorrClusEnergy(float LCorrE);
> >      const std::vector<float>& LCorrClusEnergy() const {return LocalCorr_;}
> >
> > -    void SetPFPhotonRes(float Res){Res=MVAResolution_;}
> > +    void SetPFPhotonRes(float Res){MVAResolution_ = Res;}
>
> Note that MVAResolution_ is never actually used, so this fix has no effect
> right now. However, it is good to make the fix to make the method work
> properly in case it is used in the future.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/14220/files/7b38b03e6e967d7dc3948f8555cd8f4c99642413#r60995735>
>